components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -381,20 +382,25 @@ end
 -- Finds the first round in where upper bracket opponents drop to the lower
 -- bracket. Returns nil if it cannot be determined unambiguously, or if the
 -- bracket is not double elimination.
-function Import._findDeRoundIndex(bracket)
-	if #bracket.sections ~= 2 then
-		return
+---@return number[]
+function Import._findBracketFirstDropdownRounds(bracket)
+	local firstDropdownRoundPerSection = {}
+	if #bracket.sections == 1 then
+		return firstDropdownRoundPerSection
 	end
 	local countsByRound = MatchGroupCoordinates.computeRawCounts(bracket)
-
-	for roundIndex = 1, #bracket.rounds do
-		local lbCount = countsByRound[roundIndex][2]
-		if lbCount == 0 then
-			return roundIndex
-		elseif lbCount > 0 then
-			return
+	for sectionIndex = 2, #bracket.sections do
```

Comment:
am on mobile, so might be wrong, but i think we can clean this up a bit by using array functions

something like
```lua
	return Array.map(Array.range(2, #bracket.sections), function(sectionIndex)
		return Array.find(Array.range(1, #bracket.rounds), function(roundIndex)
			return countsByRound[roundIndex][sectionIndex] == 0
		end) or -1 end)
```